app/Http/Requests/UpdateUserRequest.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace App\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class UpdateUserRequest extends FormRequest
+{
+    public function authorize()
+    {
+        return true;
+    }
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function rules()
+    {
+        $userId = $this->route('user');
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Squiz.WhiteSpace.LanguageConstructSpacing.Incorrect</br>**Severity**: error</br>**File**: app/Http/Requests/UpdateUserRequest.php</br>**Line**: 22</br>Language constructs must be followed by a single space; expected "return [" but found "return["